docs/development/testguide.rst

```diff
@@ -895,3 +895,25 @@ Python `float` objects and compared numerically.  This means that small
 differences in representation of roundoff digits will be ignored by the
 doctest.  The values are otherwise compared exactly, so more significant
 (albeit possibly small) differences will still be caught by these tests.
+
+
+Continuous integration
+----------------------
+
+Astropy uses `Travis <https://travis-ci.org/astropy/astropy>`_ for continuous
+integration on Linux and OSX setups, and `Appveyor
+<https://ci.appveyor.com/project/Astropy/astropy>`_ on Windows. These
+continuously test the package for each commit and pull request to notice
```

Comment:
Maybe "each commit and pull request that is pushed up to GitHub"?  I've noticed some novices confuse travis/appveyor with `py.test` and aren't sure which is the local thing and which is the web service...